### PR TITLE
COMMERCE-1113 Implement rowChecker to disable checkbox for System User Segments

### PR DESCRIPTION
**Pull request by alecsloan**
Let me know if you have any questions.

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

File: commerce-user-segment-web/src/main/java/com/liferay/commerce/user/segment/web/internal/search/CommerceUserSegmentEntryChecker.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	public CommerceUserSegmentEntryChecker(PortletResponse portletResponse) {
+		super(portletResponse);
+	}
+
+	@Override
+	public boolean isDisabled(Object obj) {
+		CommerceUserSegmentEntry commerceUserSegmentEntry =
+			(CommerceUserSegmentEntry)obj;
+
+		try {
```
Comment (ethanbustad) [inline commerce-user-segment-web/src/main/java/com/liferay/commerce/user/segment/web/internal/search/CommerceUserSegmentEntryChecker.java]: pretty sure the try/catch is unnecessary here

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

**Pull request merged**